src/types/schema.ts

```diff
@@ -35,6 +35,7 @@ export class PoolCreated extends Entity {
     this.set('totalAmountFunded', Value.fromBigInt(BigInt.zero()))
```

Comment:
please remove this file from the commit (this is automatically generated on build)